### PR TITLE
Removing sass

### PR DESCRIPTION
**Pull request by adrianmg**
- Removed sass dependencies
- Fixed some vulnerabilities
- Removed nesting from CSS
- Cleanup some unused resources

### CODE REVIEW

**Comment by IvanGuardado** [comment]
There are conflicts with the base branch. Please, fix them so I can do the review :)

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
+    this.state = {
+      theme : '' // example: use 'monkey' or 'minimal'
+    }
+  }
+
+  componentDidMount() {
+    // Checks for theme and loads if needed
+    if (this.state.theme) {
+      try {
+        require(`./css/themes/${this.state.theme}.css`);
```
Comment (IvanGuardado) [inline src/App.js:18]: Using require in the middle of the code is not a good practice. Let's figure out how to do this better

**Review by IvanGuardado — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (adrianmg) [reply src/App.js:18]: What's the issue exactly? Is it the issue using 'require' as an instruction or the fact we're loading resources in the middle of the code?

Comment (adrianmg) [reply src/App.js:18]: Also, would it be better to require that conditionally in the constructor? Any resources?

Comment (adrianmg) [reply src/App.js:18]: I'm going ahead, merging this and open a new branch to refactor and include react-helmet to handle this and a few more things :)

**Pull request merged**